deploy-board/deploy_board/templates/clusters/cluster_configuration.html

```diff
@@ -718,7 +718,8 @@ <h4 class="panel-title pull-left pointer-cursor">
                     var scope = this;
                     var provider = capacitySetting.currentProvider;
                     value = 'cmp_base-ebs';
-                    if (this.selectedHostTypeValue.startsWith('EbsCompute')) {
+                    const selectedHostTypeObject = capacityCreationInfo.hostTypes.find(hostType => hostType.id === this.selectedHostTypeValue);
+                    if (selectedHostTypeObject.abstract_name.startsWith('EbsCompute')) {
```

Comment:
This existing check itself does not look good to me. If user does not create host type with EbsCompute prefix, this would not work.
Is there any other property we can use to check if host is ebs/non-ebs ?